src/compiler/crystal/macros/methods.cr

```diff
@@ -1077,6 +1082,22 @@ module Crystal
 
           value
         end
+      when "has_key?"
+        interpret_check_args do |key|
+          case key
+          when SymbolLiteral
+            key = key.value
+          when MacroId
+            key = key.value
+          when StringLiteral
+            key = key.value
+          else
+            raise "argument to 'NamedTupleLiteral#has_key?' must be a symbol or string, not #{key.class_desc}:\n\n#{key}"
```

Comment:
My suggestion:

```suggestion
            raise "expected 'NamedTupleLiteral#has_key?' first argument to be a SymbolLiteral or StringLiteral, not #{key.class_desc}"

```

Reply:
i also added the `MacroId` to the list since it's in the case above